pkg/frontend/frontend.go

```diff
@@ -210,10 +210,10 @@ func (f *frontend) authenticatedRoutes(r *mux.Router) {
 	s.Methods(http.MethodPost).HandlerFunc(f.postAdminOpenShiftClusterMustGather).Name("postAdminOpenShiftClusterMustGather")
 
 	s = r.
-		Path("/admin/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/restartvm").
-		Subrouter() // remove after Geneva actions have been updated
+		Path("/admin/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}/serialconsole").
+		Subrouter()
 
-	s.Methods(http.MethodPost).HandlerFunc(f.postAdminOpenShiftClusterRedeployVM).Name("postAdminOpenShiftClusterRestartVM") // remove after Geneva actions have been updated
+	s.Methods(http.MethodPost).HandlerFunc(f.postAdminOpenShiftClusterSerialConsole).Name("postAdminOpenShiftClusterSerialConsole")
```

Comment:
Ah, now that we've replaced the "restartVM" action with "redeployVM", this can be removed?

Reply:
@ehashman Yep, confirmed it with @jim-minter